docs/devguide/repl.rst

```diff
@@ -4,42 +4,90 @@
 Accessing the REPL
 ==================
 
-Accessing the REPL on the micro:bit requires:
+REPL (Read-Evaluate-Print-Loop) allows the micro:bit to read and evaluate code 
+in real-time as you write it.
 
-    * Using a serial communication program
-    * Determining the communication port identifier for the micro:bit
-    * Establishing communication with the correct settings for your computer
+Accessing the REPL on the micro:bit will require you to:
 
-If you are a Windows user you'll need to install the correct drivers. The
+    * Determine the communication port identifier for the micro:bit
+    * Use a program to establish communication with the device
+
+If you are a Windows user you'll need to install the correct drivers, the
 instructions for which are found here:
 
 https://developer.mbed.org/handbook/Windows-serial-configuration
```

Comment:
We can update this link to: 
https://os.mbed.com/docs/latest/tutorials/windows-serial-driver.html